content/workflows/publishing-a-new-package-on-opam/index.md

```diff
@@ -18,8 +18,98 @@ tools:
+
+In order to publish to the opam repository you need to tag your project - this means adding a **tag** to indicate that a particular commit represents your piece of software at a particular version. The OCaml community strive to use [semantic versioning](https://semver.org/).
+
+![A diagram showing a git tree with the latest main branch commit tagged](/images/git-tag.png)
+
+Tagging is used in order to create a *release* of your package. A release is an [archive](https://en.wikipedia.org/wiki/Tar_(computing)) of the source code. The most common place to do this is on [Github](https://github.blog/2013-07-02-release-your-software/). You specify where a package is released to in the `url` field of the opam-repository opam files (not in the source code version as they change because of the checksum that is also added).
+
+How you tag, produce the archive, generate the slightly different opam file and create a pull-request to the opam-repository differs depending on the tool you use. Skip ahead to [the dune-release workflow](#for-dune-users) or the [opam-publish workflow](#for-everyone-else) if you don't want some information on the continuous integration opam offers.
+
+### Opam Continuous Integration 
+
+Whenever you make a pull-request to add your latest package information to the central opam repository a series of continuous integration tools are run to check your code builds, installs and how it impacts reverse dependencies (those tools that depend on your package). [TravisCI](https://travis-ci.com/) is used to check if your new release will install on a variety of platforms including MacOS, FreeBSD and arm64. [Camelus](https://github.com/ocaml-opam/Camelus) reports on a variety of problems to help ease the burden on maintainers like linting the file and checking what new packages have become available (or not) as a result of the PR. 
+
+Finally, a tool called [DataKit-CI](https://github.com/moby/datakit/tree/master/ci) does the heavy lifting and checks more platforms and the reverse dependencies. For those interested, there is a new CI tool based on [ocurrent](https://github.com/ocurrent/ocurrent) pipelines [coming soon](https://www.youtube.com/watch?v=HjcCUZ9i-ug).
+
 ## Recommended Workflow
 
-## Alternatives
+### For Dune Users 
+
+[Dune-release](/platform/dune-release) is a the recommended tool for people using dune and opam. Provided you follow some conventions, most of the hard work is taken care of by `dune-release`. To install it, simply run `opam install dune-release`. 
```

Comment:
As mentioned in the next section, maybe it should be said it's for people using dune and having their package hosted on github. There are ways to make it work if the dev-repo and so on are located elsewhere than github -- with delegates -- but it's not really the recommended way, and it's now deprecated

Reply:
Thanks 👍